ChangeLog.md

```diff
@@ -1,3 +1,26 @@
+# Changes in 1.1.8
+
+* [params.h](include/clap/ext/params.h): document how persisting parameter values between sessions should be implemented
+* [state.h](include/clap/ext/state.h): add basic documentation regarding what state should be saved and how plugins should interact with buffers
+* various documentation fixes (essentially typos)
+
+## Draft extensions
+
+* [extensible-audio-ports.h](include/clap/ext/draft/extensible-audio-ports.h): new extension which lets the host add ports to a plugin
+* [configurable-audio-ports.h](include/clap/ext/draft/configurable-audio-ports.h): new exetension which lets the host **push** an audio ports configuration request, this leads to a simpler workflow for surround host and plugins
```

Comment:
```suggestion
* [configurable-audio-ports.h](include/clap/ext/draft/configurable-audio-ports.h): new extension which lets the host **push** an audio ports configuration request, this leads to a simpler workflow for surround host and plugins
```

Reply:
If you're going to correct the typos, then change it to 

```suggestion
* [configurable-audio-ports.h](include/clap/ext/draft/configurable-audio-ports.h): new extension allowing the host to **push** an audio ports configuration request, resulting in a simpler workflow for surround host and plugins
```

:wink: 

Or 'that lets'. 'Which' is not correct here, and the 'this leads' part either needs to be its own sentence or be rewritten without 'this'.